client/src/index.jsx

```diff
@@ -1,12 +1,163 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './styles.css';
+
+class App extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {number: 1};
+		this.addSighting = this.addSighting.bind(this);
+	}
+
+	renderSightings(){
+		const {number} = this.state;
+		if (!number | number < 0) return null;
+		let total = []
+		for (let i = 0; i < number; i+=1) {
+			total.push(
+				<SightingList user={person} sighting={sight} />
+			);
+		};
+		return total;
+	};
+
+	addSighting() {
+		console.log("Clicked Sighting");
+		this.setState(state => ({
+			number: state.number + 1
+		}));
+	}
+
+	render() {
+		return (
+			<div>
+				<h1>Ahmic Animals</h1>
+				<Header addSighting={this.addSighting} />
+				<h1>MAP GOES HERE</h1>
+				{this.renderSightings()}
+			</div>
+		);
+	};
+}		
+
+
+
+
+//Nav bar at top of page
+class Header extends React.Component {
```

Comment:
This file is way too big. Ideally, you only should have one class per file

Reply:
I can add extra .jsx files, and import into index.jsx. What is the notation for importing a file? import Header from header(.jsx)?

Reply:
yeah, same way you'd import/export anything else

Reply:
OK. Pulled Header out into it's own file. Going to be lots of import calls - does it makes sense to create a new folder with exported components and then import the folder?

Reply:
A bunch of imports yes, a folder for components is a good idea, but you should only import what you're going to be using